assets/dev/scss/frontend/_container.scss

```diff
@@ -280,6 +280,13 @@
 			max-width: 100%;
 		}
 	}
+
+	.elementor-widget {
+	
+		&:not(:last-child) {
+			--kit-widget-spacing: 0px;
```

Comment:
I think this should always be 0px. We have always overwritten this styling inside the container styling to be 0px.

<img width="366" alt="Screenshot 2024-10-18 at 20 25 31" src="https://github.com/user-attachments/assets/99e70f9e-99ba-44c9-995b-fe6a19e2b2ae">

Reply:
I think your styling will always be overwritten by this style; https://github.com/elementor/elementor/blob/13ec9e53b81833fa27abc29c7c94a753c8686248/assets/dev/scss/frontend/_container.scss#L187-L186

Reply:
> Why do we have this inside the container styling?
@hein-obox Due to [this change](https://github.com/elementor/elementor/pull/29011/files#diff-5d1d967894ca0a0f77cd10dbf57faa33d3b9d62bdd118633f223a032eda43f80R17), we need to zero out the variable on containers, otherwise the container will get extra space. But we needed to add this calculation so that on Sections the margin control and kit spacing sums together.
(The margin control does not know if the widget is inside a section or container)

Reply:
> I think your styling will always be overwritten by this style;

Also not, because the margin control, like I sent in the link above, will override this. That's why we need  zero the variable specifically here.